src/routes/Settings/Settings.tsx

```diff
@@ -137,6 +141,35 @@ export const Settings = () => {
             </Stack>
           </CardContent>
         </Card>
+        <Card>
```

Comment:
It looks a bit odd to me to export the private key as a downloadable file with only one item in JSON array. I would just show it with copy button near it. Maybe we can show both eth and ed private keys and let user to copy them.

Reply:
I think some UI/UX help migh be needed here